test/functional/shada/registers_spec.lua

```diff
@@ -128,36 +128,25 @@ describe('ShaDa support code', function()
     eq({{}, ''}, getreg('h'))
   end)
 
-  it('dumps and loads register correctly when &encoding is not UTF-8',
+  it('dumps and loads register correctly with utf-8 contents',
   function()
-    set_additional_cmd('set encoding=latin1')
     reset()
     -- \171 is U+00AB LEFT-POINTING DOUBLE ANGLE QUOTATION MARK in latin1
```

Comment:
Same as with [this](https://github.com/neovim/neovim/pull/2905/files#r69023041). I believe the comment should be removed.